util/download_sign_android.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Install gh
+# Login with gh auth login
+#
+# Usage: ./download_sign_android.sh 5.13.0
+
+set -e
+
+TAG="$1"
+REPO="luanti-org/luanti"
+WORKFLOW_NAME="android"
+APKSIGNER="$HOME/Android/Sdk/build-tools/35.0.0/apksigner"
+
+if [[ -z "$TAG" ]]; then
+  echo "Usage: $0 <tag>"
+  exit 1
+fi
+
+echo "🔍 Finding workflow run for '$WORKFLOW_NAME' with tag '$TAG'..."
```

Comment:
emojis in shell scripts have a very macbook starbucks hipster feel, which is not a good fit for Luanti.